#!/usr/bin/env python
PACKAGE = "dp_hover"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("minimum_distance", double_t, 0, "Distance from hover target to slow to 0, in m",              1.0, 0.0, 50.0)
gen.add("maximum_distance", double_t, 0, "Distance from hover target where max speed is used, in m",  75.0, 0.0, 250.0)
gen.add("maximum_speed",    double_t, 0, "Maximum speed to use to approach target, in m/s",           10.0, 0.0, 30.0)
gen.add("maximum_angular_speed", double_t, 0, "Maximum angular speed, in rad/s",                       0.3, 0.0, 5.0)

exit(gen.generate(PACKAGE, "dp_hover", "dp_hover"))
